### PR TITLE
Create Rule_11-11.md

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: docs/section11/Rule_11-11.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - set a variable result equal to FAIL: `result = PASS`
+  - check if shw_bat_dict_b[shw_bat] is a dictionary.  If it's not, there is no entry (no equipment) for this shw_bat.  If it is, we need to make sure that there is no shw equipment of any type (distribution, equipment, pumps, etc) connected to the shw_bat: `if type(shw_bat_dict_b[shw_bat]) == 'dict':`
+    - look at each of the different types of equipment (distribution, equipment, pumps, loops etc): `for shw_equip_type in shw_bat_dict_b[shw_bat]:`
+      - for each of these, there should be a list.  If the list length is greater than 0, there is equipment here and we set result to FAIL: `if len(shw_bat_dict_b[shw_bat][shw_equip_type])> 0: result = FAIL`
+
+
+## Rule Assertion: 
+- Case1: result is pass: PASS: `if result == PASS: PASS`
+- Case2: all other cases is a fail: `else: FAIL`
+
```
Comment (claperle) [inline docs/section11/Rule_11-11.md:41]: This RDS appears sound if the functions have the correct names.

File: docs/section11/Rule_11-11.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Service_Water_Heating - Rule 11-11  
+**Schema Version:** 0.0.36
```
Comment (JacksonJ-KC) [inline docs/section11/Rule_11-11.md:2]: most recent version is 0.0.37

File: docs/section11/Rule_11-11.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+**Evaluation Context:** B-RMD each SWH BAT
+**Data Lookup:**   
+
+**Function Call:** 
+- **get_component_by_id**
+- **get_SHW_equipment_associated_with_each_SWH_bat**  
+- **get_SHW_types_and_SHW_use**
+
+**Applicability Checks:**
+- Each SHW use type is applicable if there are NO SHW loads
```
Comment (JacksonJ-KC) [inline docs/section11/Rule_11-11.md:23]: do we care to be more specific about checking if `ServiceWaterHeatingUse.use == 0`?

File: docs/section11/Rule_11-11.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+- call the function get_SHW_equipment_associated_with_each_SWH_bat for the baseline: `shw_bat_dict_b = get_SHW_equipment_associated_with_each_SWH_bat(B_RMD)`
+- call the function get_SHW_types_and_SHW_use for the proposed: `shw_bat_and_use_dict_p = get_SHW_types_and_SHW_use(P_RMD)`
+- look through each of the SHW bats in the building: `for shw_bat in shw_bat_and_use_dict_p:`
+  - check if there are SHW loads in this use bat, not applicable: `if len(shw_bat_and_use_dict_p[shw_bat]) > 0: NOT_APPLICABLE`
+  - otherwise, continue to rule logic for this shw_bat: `else: CONTINUE TO RULE LOGIC`
+
+  ## Rule Logic: 
+  - check that there is either NO ENTRY in shw_bat_dict_b for the shw_bat OR that for each item in shw_bat_dict_b[shw_bat] that there are 0 items in the list:
+  - set a variable result equal to FAIL: `result = PASS`
```
Comment (JacksonJ-KC) [inline docs/section11/Rule_11-11.md:32]: pseudocode and plain-english don't agree

File: docs/section11/Rule_11-11.md (hunk 3 above)
Comment (JacksonJ-KC) [inline docs/section11/Rule_11-11.md:23]: ~~SHW use type~~ SWH BAT

File: docs/section11/Rule_11-11.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# Service_Water_Heating - Rule 11-11  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-11  
+
+**Rule Description:** For buildings that will have no service water-heating loads, no service water-heating shall be modeled in baseline building model
+
+**Rule Assertion:** Options are PASS/FAIL
```
Comment (claperle) [inline docs/section11/Rule_11-11.md:10]: Shoudln't this be PASS/FAIL/IN_APPLICABLE (for buildings with SWH loads)

File: docs/section11/Rule_11-11.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+- **get_SWH_uses_associated_with_each_building_segment**  
+
+**Applicability Checks:**
+
+- look at each building segment in the proposed model: `for building_segment in B_RMD.building.building_segments:`
+    - create a boolean has_swh_loads and set it to false: `has_swh_loads = false`
+    - get the service water heating uses in the proposed building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+    - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+        - get the swh_use using get_component_by_ID: `swh_use = get_component_by_ID(P_RMD, swh_use_id)`
+        - check to see if the swh use is greater than 0 (ie there is any amount of hot water used): `if swh_use.use > 0:`
```
Comment (claperle) [inline docs/section11/Rule_11-11.md:30]: I know you have been checking if the .use is >0 to see if there are SWH loads but wonder if it is also necessary to make sure the schedule is not 0 for every hour. Maybe that is overkill. I don't feel strongly.

File: docs/section11/Rule_11-11.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            - set has_swh_loads to true: `has_swh_loads = true`
+
+    - if we arrive here with has_swh_loads still equal to false, CONTINUE TO RULE LOGIC: `CONTINUE TO RULE LOGIC`
```
Comment (claperle) [inline docs/section11/Rule_11-11.md:33]: Modify to also say that the rule is inapplicable if has_swh_loads equals true.

File: docs/section11/Rule_11-11.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+- Each SWH use type is applicable if there are NO SWH loads
+- call the function get_SWH_equipment_associated_with_each_SWH_bat for the baseline: `swh_bat_dict_b = get_SWH_equipment_associated_with_each_SWH_bat(B_RMD)`
```
Comment (claperle) [inline docs/section11/Rule_11-11.md:36]: Not following the purpose of this line

File: docs/section11/Rule_11-11.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+- call the function get_SWH_types_and_SWH_use for the proposed: `swh_bat_and_use_dict_p = get_SWH_types_and_SWH_use(P_RMD)`
+- look through each of the SWH bats in the building: `for swh_bat in swh_bat_and_use_dict_p:`
+  - check if there are SWH loads in this use bat, not applicable: `if len(swh_bat_and_use_dict_p[swh_bat]) > 0: NOT_APPLICABLE`
```
Comment (claperle) [inline docs/section11/Rule_11-11.md:39]: I am not following the reason for this section. Seems you are checking to see if any swh_uses are greater than 0 in the above logic already. Can you help me understand the purpose of this?

File: docs/section11/Rule_11-11.md (hunk 8 above)
Comment (KarenWGard) [reply docs/section11/Rule_11-11.md:36]: it was a duplicate of old logic.  Deleted

File: docs/section11/Rule_11-11.md (hunk 9 above)
Comment (KarenWGard) [reply docs/section11/Rule_11-11.md:39]: it was a duplicate that didn't get deleted.

File: docs/section11/Rule_11-11.md (hunk 6 above)
Comment (KarenWGard) [reply docs/section11/Rule_11-11.md:30]: I think that's overkill

**Review by claperle — APPROVED** [review]

File: docs/section11/Rule_11-11.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+    ## Rule Logic: 
+    - look at the same building segment in the baseline model, and make sure that there are no SWH loads in the baseline model
+    - create a boolean has_swh_loads and set it to false: `has_swh_loads_b = false`
+    - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(B_RMD, building_segment.id)`
+    - look at each service water heating use id: `for swh_use_id_b in service_water_heating_use_ids:`
+        - get the swh_use_b using get_component_by_ID: `swh_use_b = get_component_by_ID(B_RMD, swh_use_id_b)`
+        - check to see if the swh use is greater than 0 (ie there is any amount of hot water used): `if swh_use_b.use > 0:`
+            - set has_swh_loads to true: `has_swh_loads_b = true`
+
+## Rule Assertion:
```
Comment (JacksonJ-KC) [inline docs/section11/Rule_11-11.md:45]: Indent to align with Rule Logic to make clear that assertion is inside the loop through building segments

**Review by JacksonJ-KC — APPROVED** [review]
Stylistic comments

**Pull request merged**